app/src/main/java/org/koitharu/kotatsu/reader/ui/ReaderInfoBarView.kt

```diff
@@ -62,8 +62,8 @@ class ReaderInfoBarView @JvmOverloads constructor(
 	init {
 		paint.strokeWidth = context.resources.resolveDp(2f)
 		val insetCorner = getSystemUiDimensionOffset("rounded_corner_content_padding")
-		val insetStart = getSystemUiDimensionOffset("status_bar_padding_start") + insetCorner
-		val insetEnd = getSystemUiDimensionOffset("status_bar_padding_end") + insetCorner
+		val insetStart = getSystemUiDimensionOffset("status_bar_padding_start", resources.resolveDp(8)) + insetCorner
+		val insetEnd = getSystemUiDimensionOffset("status_bar_padding_end", resources.resolveDp(8)) + insetCorner
```

Comment:
It is better to use `dimen` resource for this value and obtain it once

```kotlin
        val fallbackInset = resources.getDimensionPixelOffset(R.dimen.reader_bar_inset_fallback)
		val insetStart = getSystemUiDimensionOffset("status_bar_padding_start", fallbackInset) + insetCorner
		val insetEnd = getSystemUiDimensionOffset("status_bar_padding_end", fallbackInset) + insetCorner
```

Also, the specified dimens `status_bar_padding_start` and `status_bar_padding_end` are missing on some device? Maybe this issue is vendor-specific or depends on Android version?

Reply:
I think it's vendor-specific issue, on my redmi 9A, these dimens is missing but on pixel emulator it's working fine, both running android 10